stdlib/gc.ml

```diff
@@ -38,6 +38,7 @@ type control = {
   mutable max_overhead : int;
   mutable stack_limit : int;
   mutable allocation_policy : int;
+  window_size : int;
```

Comment:
The others are mutable because when this record was introduced, the record update  feature `{ r with lbl = e }` didn't exist. You would `get`, mutate the result, and use it to `set`. Now you simply `set {(get()) with window_size = 5}`.

If I could add a "deprecated" annotation on these `mutable` flags, I would. Since it is deprecated, I see no reason to duplicate it in new fields. I can change it if you think it's too weird.